### PR TITLE
Fakelag and basic desync aa

### PR DESCRIPTION
**Pull request by x04**
It says this in the commit description, but read fakelag.h it is not the most proper way to do it but it works.

### CODE REVIEW

**Comment by x04** [comment]
I just realized a major mistake I made, in the fakelag it checks for antiaim being enabled and fakelag off. It never returns if antiaim is enabled with 1 tick choke enabled. But that's a small fix, you can do it immediately after merging the PR if you do merge it.

**Comment by alpine971** [comment]
Did you test fakelag?

**Comment by x04** [comment]
> Did you test fakelag?

Yes, it's working, just read my comment and what I said about using m_choked in the fakelag.h

File: csgo/features/anti-aim/antiaim.cpp [hunk 1]
```diff
@@ -37,40 +31,37 @@ float at_target( ) {
...
 	return best_angles.y;
 }
 
-float c_antiaim::calculate_max_desync_angle( C_CSPlayer *local, CCSGOPlayerAnimState *anim_state ) {
-	return 0.f;
-}
-
 void c_antiaim::adjust_yaw( CUserCmd *ucmd ) {
-
+	if ( g_cl.m_sendpacket ) {
```
Comment (alpine971) [inline csgo/features/anti-aim/antiaim.cpp]: I don't think that this is how desync works.

**Comment by alpine971** [comment]
Just make a separate pull request with only the fakelag and i’ll merge it

**Comment by x04** [comment]
> Just make a separate pull request with only the fakelag and i’ll merge it

I updated this one.

**Pull request merged**